packages/cmrjs/cmr-utils.js

```diff
@@ -666,7 +698,8 @@ async function updateCMRMetadata({
   files,
   distEndpoint,
   published,
-  inBuckets = null
+  inBuckets = null,
+  cmrGranuleUrlType = 'distribution'
```

Comment:
I think the `task/move-granules/index.js` file should be updated https://github.com/nasa/cumulus/blob/master/tasks/move-granules/index.js#L238 to use the newly added `cmrGranuleUrlType` value.